fusestore/store/src/configs/config.rs

```diff
@@ -2,10 +2,24 @@
 //
 // SPDX-License-Identifier: Apache-2.0.
 
+use lazy_static::lazy_static;
 use structopt::StructOpt;
 use structopt_toml::StructOptToml;
 
-pub const FUSE_COMMIT_VERSION: &str = env!("FUSE_COMMIT_VERSION");
+lazy_static! {
```

Comment:
Cool.
The lazy looks duplicated with fusequery/query/src/configs/config.rs

Reply:
Aye sir.
I left these boilerplate there because I'm not very sure about if it is all right for two binaries sharing a common version.
Moving these to a common lib would be better?

Reply:
+1

Reply:
@BohuTANG 
There is an issue with sharing these common codes:
The `option_env` must be evaluated after `build.rs` finished.
`build.rs` depends on `common/building`. Thus `common/building` will be compiled without knowing of these env vars.
Then to share the common codes, they must be moved to another crate that is not `common/building`.

For now, no other `common` is suitable for this task.
Thus we have to: 
- Let `fusestore` reference `fusequery::*::FUSE_COMMIT_VERSION` or reverse.
- or create another crate to keep the common codes.

Reply:
Got